automation/test.sh

```diff
@@ -380,6 +380,7 @@ spec:
     server: "nfs"
     path: /
   storageClassName: windows
+  persistentVolumeReclaimPolicy: Recycle
```

Comment:
@xpivarc unfortunately recycle spins up a pod that `rm -rf`'s the data on the PV so this won't work

Reply:
/close